src/core/image.js

```diff
@@ -705,7 +705,7 @@ class PDFImage {
       isOffscreenCanvasSupported &&
       ImageResizer.needsToBeResized(drawWidth, drawHeight);
 
-    if (this.colorSpace.name === "DeviceRGBA") {
+    if (!this.mask && this.colorSpace.name === "DeviceRGBA") {
```

Comment:
Should this perhaps ignore all kinds of masks?
```suggestion
    if (!this.smask && !this.mask && this.colorSpace.name === "DeviceRGBA") {
```